src/language/CodeInspection.js

```diff
@@ -199,6 +199,21 @@ define(function (require, exports, module) {
     }
 
     /**
+     * Returns an array of the names of providers registered for a specific language
+     *
+     * @param {!string} languageId
+     * @return {Array.<string>} Names of registered providers.
+     */
+    function getProviderNamesForLanguage(languageId) {
```

Comment:
I believe it's not implemented yet, but still... This method should be called `getProviderIDsForLanguage()`. Currently name of the provider serves as its ID, but this may change in the future. The difference being `id` would be something to locate the provider, e.g. put it in the pref file and `name` would be something to display. It's OK to return `provider.name` in this function, but the signature should mention ID instead of name.